perun-core/src/test/java/cz/metacentrum/perun/core/impl/modules/pwdmgr/MuadmPasswordManagerModuleTest.java

```diff
@@ -0,0 +1,62 @@
+package cz.metacentrum.perun.core.impl.modules.pwdmgr;
+
+import cz.metacentrum.perun.core.AbstractPerunIntegrationTest;
+import cz.metacentrum.perun.core.api.exceptions.PasswordStrengthException;
+import cz.metacentrum.perun.core.bl.PerunBl;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.fail;
+
+public class MuadmPasswordManagerModuleTest extends AbstractPerunIntegrationTest {
+
+	private MuadmPasswordManagerModule module;
+
+	@Before
+	public void setUp() {
+		this.module = (MuadmPasswordManagerModule) ((PerunBl) sess.getPerun()).getUsersManagerBl()
+			.getPasswordManagerModule(sess, "mu-adm");
+	}
+
+	@Test
+	public void weakPasswordsTest() {
+		List<String> weakPasswords = List.of(
+			"eakfjklsjweiml",
+			"LOSFNXHRKJANCS",
+			"18746982365414",
+			"!@#$%^&*_-+=?/",
+			"AkoHCNqrXMsuqm",
+			"afgr487xiy2m63",
+			"!lop@#twoep&*/",
+			"RT672HWZ1068KK",
+			"A!@#KDSAJ?/NMO",
+			"265?!8+789#_=^"
+		);
+
+		weakPasswords.forEach(password -> assertThrows(PasswordStrengthException.class, () -> {
+			module.checkPasswordStrength(sess, "login", password);
+		}));
+	}
+
+	@Test
+	public void strongPasswordsTest() {
+		List<String> strongPasswords = List.of(
+			"dskglJDOK46854",
+			"bcvjoEWIQO!@#$",
+			"viwuq24736/?+-",
+			"WMKCO03877@#/*",
+			"jfsUOIW5798+-*"
+		);
+
+		strongPasswords.forEach(password -> {
+			try {
+				module.checkPasswordStrength(sess, "login", password);
+			} catch (PasswordStrengthException e) {
+				fail();
+			}
```

Comment:
If you want to test that some exception is thrown, I suggest using the `assertThatExceptionOfType()` method from the assertJ library. For inspiration, you can take a look at `urn_perun_vo_attribute_def_def_applicationExpirationRulesTest` where this method is already used.